ephios/api/views/events.py

```diff
@@ -96,3 +96,11 @@ class EventViewSet(viewsets.ReadOnlyModelViewSet):
         .prefetch_related(Prefetch("shifts__participations"))
         .order_by("start_time")
     )
+
+
+class ParticipationSerializer(serializers.ModelSerializer):
+    state = serializers.CharField(source="get_state_display")
```

Comment:
This should not be the translated state name. I'd say lets use the integer ID of the state and the english codeword optionally as well.